### ISSUE TITLE
Default `errorLevel` value from XSD file is ignored

### ISSUE THREAD

**Issue by remorhaz** [vimeo/psalm#3775]
When I set `errorLevel` to `1` in my `psalm.xml.dist`, it works okay, but my editor tags this attribute as redundant (which is okay because `config.xsd` contains `1` as default value). But when I remove it, Psalm runs with greater level (some errors are reported as "other issues").

Shouldn't the default error level be the same in XSD and in utility?

**Comment by remorhaz** [comment]
No, it's a CLI problem.

**Comment by orklah** [comment]
Please check if you have totallyTyped or reportMixedIssues attribute in your file and delete them if they're here

those 3 attributes have a weird overlap on lower levels

**Comment by remorhaz** [comment]
No. My settings is pretty basic:
```xml
<?xml version="1.0"?>
<psalm
        resolveFromConfigFile="true"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xmlns="https://getpsalm.org/schema/config"
        xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd">
    <projectFiles>
        <directory name="src"/>
        <ignoreFiles>
            <directory name="vendor"/>
        </ignoreFiles>
    </projectFiles>
</psalm>
```

**Comment by orklah** [comment]
Ok, if you put reportMixedIssues to false, it should allow you to remove errorLevel while keeping Mixed issues from popping

**Comment by remorhaz** [comment]
But I want them to pop! I mean, I _do want_ to have errorLevel 1 and want to have all errors popping; but I found it weird that setting the default value explicitly changes the behavior.

**Comment by weirdan** [comment]
If xsd says default is 1 omitting the attribute should behave as if it was set to 1.

**Comment by micheh** [comment]
@weirdan The level is set to 2 if neither the `errorLevel` nor `totallyTyped` is specified in the configuration:
https://github.com/vimeo/psalm/blob/46013a35c84a0d547e0167b87e3a7da4f045cd8d/src/Psalm/Config.php#L934

Shouldn't the default in the xsd be updated to 2?

**Comment by gharlan** [comment]
default value in xsd is `2` now: https://github.com/vimeo/psalm/pull/5339
so i think this can be closed.
